### ISSUE TITLE
Default fleet endpoint incorrect if docker service IP changes (flanneld)

### ISSUE THREAD

**Issue by sgrimee** [yodlr/CoreGI#23]
The default values:
```
FLEET_ENDPOINT = 'http://172.17.42.1:4001'
ETCD_HOST = '172.17.42.1
```
will not work if the ip address of the docker service is changed, which is the case when flanneld is used.

I will submit a PR that sets it to the host private ip in the unit file, if you think it is a correct fix.

**Comment by jareddlc** [comment]
I'll take a look at the PR when I have time.

**Comment by rosskukulinski** [comment]
Fixed in #24
